scripts/projects.js

```diff
@@ -0,0 +1,9 @@
+var selectedProjects = [
+  {
+    title:  'BirdFeed',
+    github: 'http://github.com/elhendricks/elhendricks.github.io',
+    description:  '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum tempus finibus suscipit. Nullam pulvinar magna erat, quis vestibulum nulla porttitor at. Ut iaculis vitae leo non tristique. In sed luctus augue. Cras quis erat posuere, sodales turpis eu, varius dolor. Aliquam dignissim pretium nisi, sollicitudin venenatis ipsum tincidunt nec. Sed finibus, nulla in tincidunt congue, nulla sapien tincidunt lacus, ac consectetur sem mauris at dui.</p>',
+    screenshot: 'src="images/birdFeed.jpg'
```

Comment:
You don't need the "src=" code in your data. This is actually breaking the img link in the resulting code.